src/assets/i18n/global/hi-global.json

```diff
@@ -276,6 +276,17 @@
 
   "SELECT_USER_TYPE": "उपयोगकर्ता प्रकार का चयन करें",
   "PRINCIPAL":"प्रधान अध्यापक",
-  "BLOCK_EDUCATIONAL_OFFICER":"प्रखंड शिक्षा पदाधिकारी"
+  "BLOCK_EDUCATIONAL_OFFICER":"प्रखंड शिक्षा पदाधिकारी",
+
+  "YOUR_SESSION_EXPIRE":"आपका सत्र समाप्त हो गया है! कृपया फिर भाग लें",
```

Comment:
@apurvaubade change this `कृपया फिर भाग लें` translation to something `फिर से लॉगिन करने का प्रयास करें`